Makefile

```diff
@@ -108,42 +108,22 @@ uninstall:
 	pip freeze | grep -v "@" | xargs pip uninstall -y
 	rm -f $(env_tag) $(env_dev_tag) $(pre_deps_tag) $(install_tag)
 
-install: setup $(install_tag)
-
-install_dev: setup_dev $(install_tag)
-
-reinstall: install
-	@echo "==Rebuilding package distribution=="
-	${PYTHON} setup.py --quiet sdist
-	ls -rt  dist/* | tail -1 > dist/.build-tag
-	@echo "==Reinstalling package=="
-	${PYTHON} -m pip install --quiet $(shell ls -rt  dist/*.tar.gz | tail -1)
-	touch $(install_tag)
-
-reinstall_dev: install_dev
-	@echo "==Rebuilding package distribution=="
-	${PYTHON} setup.py --quiet sdist
-	ls -rt  dist/* | tail -1 > dist/.build-tag
-	@echo "==Reinstalling package=="
-	${PYTHON} -m pip install --quiet $(shell ls -rt  dist/*.tar.gz | tail -1)
-	touch $(install_tag)
-
-sort_imports: setup_dev
-	${PYTHON} -m isort $(folders)
+install: $(install_tag)
```

Comment:
Do we really need the `$(install_tag)` command? Couldn't we simply put its whole logic inside this command?

Reply:
No, I actually meant the whole `$(install_tag)` make command:
```
install: dist/.build-tag
	@echo "==Installing package=="
	${PYTHON} -m pip install --quiet --no-deps $(shell ls -rt  dist/*.tar.gz | tail -1)
	touch $(install_tag)
```
instead of
```
$(install_tag): dist/.build-tag
	@echo "==Installing package=="
	${PYTHON} -m pip install --quiet --no-deps $(shell ls -rt  dist/*.tar.gz | tail -1)
	touch $(install_tag)

install: $(install_tag)
```

But I realised that we actually cannot do this since this way `make` wouldn't have a reference file to check to avoid reinstalling the package.